static/js/gio_calendar_next_cm.js

```diff
@@ -19,23 +19,33 @@ function retrieveNextCommunityMeeting() {
       // check if various properties exist, and set variables if so
       var timeRange = e.pretty_time ? `<time><i class="fa fa-time"></i> ${e.pretty_time}</time><br>` : '';
       var shortDescription = e.short_description ? `<p>${e.short_description}</p>` : '';
-      var generalLocation = e.location ? `<location><strong>Location:</strong><br>${e.location}</location><hr>` : '';
-      var eventLink = e.event_url ? `<p><a href="${e.event_url}"><i class="fa fa-new-window"></i> Event Link</a></p>` : '';
+      var generalLocation = e.location ? `<location><strong>Location:</strong><br>${e.location}</location>` : '';
+      var eventLink = e.event_url ? `<hr><button class="event-link">
+        <span class="circle" aria-hidden="true">
+          <span class="icon arrow"></span>
+        </span>
+        <span class="button-text"><p><a href="${e.event_url}" target="_blank"><i class="fa fa-new-window"></i> <span class="button-text">Event Link</span></button></a></p>` : '';
       var communityMeetingAgenda = e.community_meeting_agenda_url ? ` <a class="btn btn-tnris btn-md" href="${e.community_meeting_agenda_url}">Download Agenda</a>` : '';
       // fill dom list element with event details
       event.innerHTML =
         `
-        <h3>
-          <strong>${e.pretty_date} <i class="fa fa-star" style="color: #1e8dc1;"></i></strong>
-          <br>
-          ${e.title}
-        </h3>
-        ${timeRange}
-        ${shortDescription}
-        ${generalLocation}
-        ${eventLink}
+      <h3>
+        <strong>${e.title}</strong>
+        <br>
+      </h3>
+      <h5>
+        ${e.pretty_date}<i class="fa fa-star" style="color: #1e8dc1;"></i>
```

Comment:
this font awesome star icon doesn't seem to be showing up that i can see

Reply:
<img width="1430" alt="Screen Shot 2021-04-28 at 1 47 11 PM" src="https://user-images.githubusercontent.com/70979203/116460053-34a22100-a82c-11eb-8cd4-201ac099027f.png">

We can't see this section now since it only shows events checked as community meetings, and the Q2 meeting was just yesterday so it no longer shows. I tested in the screenshot above by taking the communty meeting requirement out of the API url and it seems to be showing